app/src/ui/history/selected-commit.tsx

```diff
@@ -276,7 +275,7 @@ export class SelectedCommit extends React.Component<
       return null
     }
 
-    return <DragOverlay dragZoneDescription="branch-button" />
+    return <div id="drag-overlay"></div>
```

Comment:
Why didn't you remove this function (and the `showDragOverlay` prop) altogether? 🤔 

Just keeping changes to a minimum

Reply:
We still want to only show the drag overly when this.props.showDragOverlay. I think I am misunderstanding you..